docs/HowToPortTizenRTOnWiFiChipset.md

```diff
@@ -0,0 +1,141 @@
+# How to port TizenRT on new WiFi chipset
+
+There are two cases to consider towards porting TizenRT on a new WiFi chipset. The specific case depends on the vendor's decision to either use their proprietary supplicant codebase, or
+to directly use TizenRT's WiFi solution.
+
+Below are mentioned the two cases in detail. Additionally, instructions are provided to incorporate the WiFi chipset driver in TizenRT's code tree. Specifically, details are provided
+to interface driver to network stack, and to get/set WiFi configuration from a higher abstraction layer.
+ 
+## How to add external WiFi library to TizenRT
+
+External vendors may choose to build TizenRT application that includes their compiled WiFi libary, rather than opting for TizenRT's *wpa_supplicant*.
+To support external and independently compiled WiFi supplicant libraries, they must be merged with *libexternal.a* static libary that resides in the *external* folder of TizenRT.
```

Comment:
This line is not needed. They don't need to merge external built library and our external library. IoTivity isn't merged to our external library either.